cmd/difference_test.go

```diff
@@ -0,0 +1,51 @@
+/*
+ * Minio Client (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"testing"
+)
+
+//TestFind is the structure used to contain params pertinent to find related tests
+type TestExclude struct {
```

Comment:
We don't need a type to be outside we can simply use 

```
testCases := []struct{}
``` 

As opaque structs, this is to avoid namespace because `TestExclude` as a struct name is just too generic.  You can move the code entirely inside the test function itself. 

Look at the master `find_test.go` for more examples on this.